inference-engine/src/vpu/graph_transformer/src/stages/mvn.cpp

```diff
@@ -88,11 +90,13 @@ void FrontEnd::parseMVN(const Model& model, const ie::CNNLayerPtr& layer, const
     for (int i = 0; i < indicesSize; i++) {
         axes.insert(getDimFromAxis(ndims, indicesPtr[i]));
     }
+    const auto width = axes.count(Dim::W);
 
-    VPU_THROW_UNLESS(!axes.count(Dim::N) && axes.count(Dim::H) && axes.count(Dim::W),
+    VPU_THROW_UNLESS(!axes.count(Dim::N) && width,
                      "Unsupported combination of indices in layer \"%s\". "
                      "Only across channel and full batch supported.", layer->name);
```

Comment:
please, fix error message according to the changes

Reply:
fixed